mm2src/mm2_main/tests/docker_tests/docker_ordermatch_tests.rs

```diff
@@ -12,7 +19,7 @@ fn check_asks_num(mm: &MarketMakerIt, base: &str, rel: &str, expected: usize) {
     })))
     .unwrap();
     assert!(rc.0.is_success(), "!orderbook: {}", rc.1);
-    let orderbook: OrderbookResponse = json::from_str(&rc.1).unwrap();
+    let orderbook: OrderbookResponse = serde_json::from_str(&rc.1).unwrap();
```

Comment:
I have done this to avoid confusion between `self as json` as crate and `json!` macro. `serde_json` is short enough to be used directly. So I prefer switching to it over entire codebase gradually.

Reply:
Ok I see, thanks!